cmd/wallet-sdk-gomobile/go.mod

```diff
@@ -53,7 +53,11 @@ require (
 	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
 	github.com/xeipuuv/gojsonschema v1.2.0 // indirect
 	golang.org/x/crypto v0.1.0 // indirect
+	golang.org/x/mobile v0.0.0-20221110043201-43a038452099 // indirect
```

Comment:
Run `go mod tidy` here to remove these entries. (I set up the bindings scripts to tidy this up automatically, but if they're interrupted partway through then this happens... this happens to me too)